src/api/article/article.model.js

```diff
@@ -159,13 +159,23 @@ export const queryUnpublished = user => {
   return {};
 };
 
+export const DEFAULT_ARTICLE_QUERY = user => ({
+  $and: [
+    {
+      active: true,
+      locales: { $exists: true },
+    },
+    queryUnpublished(user),
+  ],
+});
+
 export const POPULATE_OPTIONS = {
   collection: user => ({
     path: 'collectionId',
     select: '-_id name slug description cover articles podcastCover',
     populate: {
       path: 'articles',
-      match: queryUnpublished(user),
+      match: DEFAULT_ARTICLE_QUERY(user),
```

Comment:
CAPS for constants, better naming here is `getDefaultArticleQueey(user)`

Reply:
Done.